wait-for-deployment

```diff
@@ -10,9 +10,14 @@
 #
 set -o errexit
 set -o pipefail
-set -o nounset
```

Comment:
See my other comment on why we don't need to disable nounset.

(And even if we needed to, a much better approach is to leave it globally enabled and just disable it in a block of choice. But again, I think we should be able to keep it enabled permanently.)